app/views/initiatives/index.html.erb

```diff
@@ -11,7 +11,7 @@
           <p class="init-description"><%= initiative.description %></p>
           <div>
             <div class="init-footer w-row">
-              <div class="w-col w-col-2" style="padding: 0;"><%= image_tag "icon-calendar.svg", class: "icon-target" %></div>
+              <div class="w-col w-col-2" style="padding: 0;"><%= image_tag "icon-#{initiative.completion ? 'completion': 'calendar'}.svg", class: "icon-target" %></div>
```

Comment:
I called this out on PR #4 as well--what's the thinking behind having inline styling here? I don't often see it used because it can be hard to track down the right place to change if you have styling at multiple levels of the application (ie, stylesheet from an external library, styling from application level CSS, and inline styling). Inline styling also isn't reusable,  so if you decide all the places you have 0 padding you now want 10px, you have to change each instance individually. Would it make sense to define this style in a class and add another class name here?

Reply:
A class has been added on the css stylesheet